### PR TITLE
Update README for PnP sample

### PR DESCRIPTION
**Pull request by momuno**
Fills in descriptions for use of Azure IoT Explorer. Orders capabilities in order encountered in sample.

### CODE REVIEW

File: sdk/samples/iot/README.md [hunk 1]
```diff
@@ -74,25 +74,59 @@ This section provides an overview of the different samples available to run and
...
+            "ac": 200,
+            "av": 14,
+            "ad": "success"
+          },
+          "maxTempSinceLastReboot": 74.3,
+      }
+  }
+  ```
+
+- **Direct Method (Command)**:  One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an ISO8062 time format and select Invoke method.
```
Comment (ahsonkhan) [inline sdk/samples/iot/README.md:111]: > using an ISO8062 time format 

What's ISO8062? Doesn't seem related:
https://www.iso.org/obp/ui/#iso:std:iso:8062:-4:ed-1:v1:en

```suggestion
- **Direct Method (Command)**:  One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an ISO8601 time format and select Invoke method.
```

File: sdk/samples/iot/README.md [hunk 2]
```diff
@@ -74,25 +74,59 @@ This section provides an overview of the different samples available to run and
...
-    "endTime": "<ISO8601 time>"
+  "properties": {
+      "desired": {
+          "targetTemperature": 68.5,
+      }
   }
   ```
 
-  with correct values substituted for each field.
+  Upon receiving a desired property message, the sample will update the twin property locally and send a reported property   of the same name back to the service.  This message will include a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.send a reported property message back to the service.
```
Comment (ahsonkhan) [inline sdk/samples/iot/README.md:95]:
```suggestion
  Upon receiving a desired property message, the sample will update the twin property locally and send a reported property of the same name back to the service.  This message will include a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.send a reported property message back to the service.
```

Comment (ahsonkhan) [reply sdk/samples/iot/README.md:95]: >  an optional `ad` for an ack description.send a reported property message back to the service.

Looks like two sentences got merged. Fix wording/spacing?

File: sdk/samples/iot/README.md (hunk 1 above)
Comment (momuno) [reply sdk/samples/iot/README.md:111]: that's because it's a typo lol

**Comment by momuno** [comment]
Azure/azure-sdk-eng please ignore.  This PR no longer references your docs.

File: sdk/samples/iot/README.md [hunk 3]
```diff
@@ -74,25 +74,59 @@ This section provides an overview of the different samples available to run and
...
+            "ac": 200,
+            "av": 14,
+            "ad": "success"
+          },
+          "maxTempSinceLastReboot": 74.3,
+      }
+  }
+  ```
+
+- **Direct Method (Command)**: One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an [ISO8061](https://en.wikipedia.org/wiki/ISO_8601) time format and select Invoke method.
```
Comment (danewalton) [inline sdk/samples/iot/README.md:111]:
```suggestion
- **Direct Method (Command)**: One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) time format and select Invoke method.
```

Comment (momuno) [reply sdk/samples/iot/README.md:111]: me and typos.  maybe I started too early, or stayed up too late on this.
